### ISSUE TITLE
Div elements created by jqgrid are flagged with accessibility issues

### ISSUE THREAD

**Issue by julie4754** [tonytomov/jqGrid#992]
Div elements created by jqgrid are created with ids but not names. As these dvs are interactive, WCAG say these should have names.

Rule: Interactive component must have a programmatically associated name

Example:

<div style="width: 570px;" id="gview_unreadInboxList" aria-multiselectable="true" role="grid" class="ui-jqgrid-view">

**Comment by tonytomov** [comment]
Hello,

Thank you for the recommendation.
